bin/aws-semaphore-agent.js

```diff
@@ -20,6 +20,10 @@ new AwsSemaphoreAgentStack(app, 'AwsSemaphoreAgentStack', {
   },
 });
 
+if (!argumentStore.isEmpty("SEMAPHORE_AGENT_TAG_NAME")) {
+  Tags.of(awsSemaphoreAgentStack).add('Name', argumentStore.get("SEMAPHORE_AGENT_TAG_NAME"));
```

Comment:
I think you'd need to require `Tags` for this

Reply:
yeah, you are right